repeatString/repeatString.js

```diff
@@ -1,5 +1,11 @@
-const repeatString = function() {
 
-}
+function repeatString(string, times) {
+    if(times < 0) 
+      return "";
+    if(times === 1) 
+      return string;
+    else 
+      return string + repeatString(string, times - 1);
+  }
```

Comment:
@maheralayoubi 

This one is wrong,

When you call
```javascript
var str = repeatString('hello', 5);
console.log(str); // <<<<< should output the below
```

It should print

```
hellohellohellohellohello
```


so you should repeat the string with the passed number.

Reply:
Okay, I will fix it

Reply:
!!!! Hint

You need to loop it bro...

@maheralayoubi